### DISCUSSION TITLE
What kind of links are not allowed?

### DISCUSSION THREAD

**Discussion by jcubic** [EddieHubCommunity/LinkFree#1350] [Q&A]
Hi, I made a PR to make a link `a` tag some time ago, which is good for SEO. But my question is what kind of links are not allowed? I guess you have strict rules and I don't want to waste my time adding a PR that will be rejected.

Here is an example of Links that I want to add:

https://linktr.ee/jcubic

If this is ok, I may add your project to my article: [How to Promote your Open Source Project with SEO](https://itnext.io/seo-for-open-source-projects-1a6b17ffeb8b).

**Comment by NS007-dev** [comment]
@jcubic you can add any type of link you wish, using this code in your json object: 

  {
      "name": "LinkTree Profile",
      "url": "https://linktr.ee/jcubic",
      "icon": "link"
    }

I hope this has helped you and resolved your question. Any further questions, feel free to ask!

**Reply by jcubic** [reply]
Do you mean that I can put links that violate your Code of Conduct? Can I put links about nudity, profanity, and hazard?

PS: you actually need to open that page that is similar to linkFree and has a bunch of links that I want to add.

**Reply by NS007-dev** [reply]
@jcubic I am clarifying that your content and links **should not** violate the code of conduct. 
Thank you.

**Reply by NS007-dev** [reply]
> PS: you actually need to open that page that is similar to linkFree and has a bunch of links that I want to add.

As long as your links do not violate with the CoC, and corrolates to the meaning and purpose of LinkFree, feel free to add them to your profile. 
Thank you.

**Reply by jcubic** [reply]
> corrolates to the meaning and purpose of LinkFre

What is it exactly? Is it written somewhere? This is exactly what my question is about.

**Reply by yaswanthteja** [reply]
> > corrolates to the meaning and purpose of LinkFre
> 
> What is it exactly? Is it written somewhere? This is exactly what my question is about.

Just go through the Code of conduct once to know more about that

**Comment by thuongtruong1009** [comment]
@jcubic !
In this repo, scroll to the newest PR list, and you can easily add some JSON lines that:
```bash
{
  "name": "You_name_profile",
  "url": "url_or_linktree",
  "icon": "url_of_icon"
}
```
* You can create any PR or anything you want, as long as they don't violate the Code of Conduct of the community. A few examples are:
   + Avoid advertising links, no spam...
   + Avoid war, avoid code conflict...
   + PR should have formats like title, name, description, and purpose....
* You can make anything you want to contribute to the community and That work is encouraged

* Hope the above helps and answers your questions. If you find this comment appropriate, please click **mark them as the answered** to help others see and understand them.
##### Thank you very much!

**Reply by jcubic** [reply]
thanks I can read the code I know what JSON looks like. I was asking about the rules not the syntax of the JSON file.

**Reply by thuongtruong1009** [reply]
@jcubic Oh, I think not to have the syntax in this repo. This means you can add anything you want, as long as it not conflict with base branch!

**Reply by thuongtruong1009** [reply]
![image](https://user-images.githubusercontent.com/71834167/174477094-f0241245-62e6-4f88-a910-9e207d1d6c6b.png)
@jcubic , it simply is you add your information to display or show on the website or README of the organization. They are considered as the first contribution to joining the organization. I simple like that

**Reply by jcubic** [reply]
@thuongtruong1009 I'm not part of the organization. You can still contribute to projects on GitHub without joining the organization.

**Reply by jcubic** [reply]
Actually, this is standard on GitHub, you usually join an organization when you have a project that you contribute a lot to and got invited to join.

**Reply by thuongtruong1009** [reply]
> Actually, this is standard on GitHub, you usually join an organization when you have a project that you contribute a lot to and got invited to join.

Oh, yes. I think so!

**Reply by thuongtruong1009** [reply]
> @thuongtruong1009 I'm not part of the organization. You can still contribute to projects on GitHub without joining the organization.

Wow, you're interesting. If you are not in the organization, it not have any constraint in your contribution by PR. You want make any thing you want. 😅
Have a nice day!

**Reply by NS007-dev** [reply]
@jcubic 

1. You can simply join EddieHub's organisation, by opening an issue here: [Issue to join EddieHub](https://github.com/EddieHubCommunity/support/issues/new?assignees=&labels=invite+me+to+the+organisation&template=invitation.yml&title=Please+invite+me+to+the+GitHub+Community+Organization) . So you do not have to keep contributing to get invited. But, if you are devoted and a regular user, you would contribute to your organisations regularly.
2. As you can see, the rules of writing code for this repo is to either fix bugs, add new features to improve the documentation or code, add your profile/ make your own LinkFree profile and help others. 
3. Pull requests that violate the Code of Conduct will be reported and closed. 
4. Pull requests that do not relate to the purpose of the repository, will be closed.
5. Read the README.d to see what you can do in each repository.

Hope this is more clear. Thank you.

**Reply by jcubic** [reply]
@CodingSpecies I don't want to join your organization I just want to know if you open the link https://linktr.ee/jcubic are links that are inside that ok. I'm not asking about that link that is in a comment.

**Reply by NS007-dev** [reply]
@jcubic all the links within your linkTree profile is ok. Yes you can use them. Thank you.
